lib/plugin.js

```diff
@@ -1,3 +1,4 @@
+let _initialized = {}
```

Comment:
TIP: hared state objects like this should be avoided because it will be shared across SSR requests.

Reply:
thank you i know about that

i was considered where can assign initialized variable, i think to try `nuxt context(ctx)` or `global` but i don't found appropriate positions.